repos/system_upgrade/common/actors/ipuworkflowconfig/libraries/ipuworkflowconfig.py

```diff
@@ -50,12 +50,16 @@ def get_os_release(path):
     try:
         with open(path) as f:
             data = dict(l.strip().split('=', 1) for l in f.readlines() if '=' in l)
+            release_id = data.get('ID', '').strip('"')
+            version_id = data.get('VERSION_ID', '').strip('"')
+            if release_id == 'centos' and '.' not in version_id:
+                version_id = "{}.999".format(version_id)
```

Comment:
I am not too happy about 999 magic constant tbh :( 
Is there a universe where version_id is '' and leapp is still happy?